### ISSUE TITLE
Broken cursor after screenshot

### ISSUE THREAD

**Issue by santilococo** [flameshot-org/flameshot#1824]
**Flameshot version**
Flameshot v0.10.1
Compiled with Qt 5.15.2

**Describe the bug**
Every time I launch the flameshot gui, my terminal cursor breaks.

**To Reproduce**
Just launch flameshot gui and copy to clipboard.

**Expected behavior**
Normal behavior: https://i.imgur.com/rsvKvRH.png
Unexpected behavior: https://i.imgur.com/tFLDpKT.png

**System Information**
Linux archLinux 5.13.9-arch1-1

**Comment by mmahmoudian** [comment]
I remember that this was mentioned in another thread but after 10 minutes of searching I couldn't find it.

Anyways, there are few more things that I need you to report as this is not reproducible on my Manjaro machine 

Tell us:
- that shell do you use (e.g bash, zsh, fish, nushell)
- what terminal application do you use and if you have the GPU processing enabled in it
- are you on X11 or Wayland (check [here](https://flameshot.org/guide/issue-reporting/) if you don't know how to)

**Comment by santilococo** [comment]
Hi!

- zsh shell
- alacritty (with GPU processing) and st terminal (dont know if it is enabled or not)
- X11

The cursor will be broken throughout the life of the terminal. If I open vim, it will be like this as well https://i.imgur.com/tFLDpKT.png and it will not change when going to normal mode or insert mode for example. It is really annoying.

**Comment by veracioux** [comment]
I have the same problem. (Arch, alacritty)

The problem seems to be  that after flameshot closes, alacritty does not receive focus again. I would have to click on another window and then on alacritty again to focus it properly.

I have a suspicion that this is caused by the window manager (i3 in my case), specifically because it is a tiling one.

@mmahmoudian @santilococo Which WM/DE do you use?

**Comment by santilococo** [comment]
Hi. I also use a tiling window manager (DWM).

**Comment by mmahmoudian** [comment]
I use KDE Plasma on one computer and AwesomeWM on the other and on both I use Alacritty and on Plasma I also use Yakauke and Konsole as well. On Plasma I definitely do not have this issue. I will re-check AwesomeWM and post here, but as far as I can recall i don't have any issue there with Alacritty either.

**Comment by santilococo** [comment]
OK. TY.

**Comment by santilococo** [comment]
@veracioux Indeed. It seems to be a 'focus' problem. If I manually focus the terminal, the cursor will fix.

**Comment by mmahmoudian** [comment]
I confirm that this is the issue with AwesomeWM and Alacritty and Konsole, 

**Alacritty:**

https://user-images.githubusercontent.com/390889/133479266-8f024520-6f98-4677-b427-26bc062ecee8.mp4

**Konsole:**

https://user-images.githubusercontent.com/390889/133479262-01db2486-caea-4e3c-b00c-1557d86fac48.mp4

**Comment by mmahmoudian** [comment]
This might be also related to #1694

**Comment by borgmanJeremy** [comment]
So this sounds like an upstream issue with awesomeWM?

**Comment by mmahmoudian** [comment]
@borgmanJeremy I don't know. If we manage to reproduce this is any other WM (other than DWM and AwesomeWM), then that proves the bug is on us I guess

**Comment by mohad12211** [comment]
#784, #1694, and this issue are all the same problem for sure. focus is not set correctly after exiting. both me and @santilococo are dwm users and confirm this issue. also I faced it on bspwm. @sebw in #784 mentioned that he used bspwm. as a start I would like people to try this [wrapper script](https://github.com/flameshot-org/flameshot/issues/784#issuecomment-918382722). you need xdotool installed. it solved the issues for me (broken cursor and chrome menus) on dwm and bspwm. maybe we can figure something out after that?

**Comment by sebw** [comment]
I have the problem in both i3wm and bspwm.
